lib/service/logger.js

```diff
@@ -24,7 +24,7 @@ function _log(level, message) {
     level
   };
 
-  logMessage = JSON.stringify(logMessage) + '\n';
+  logMessage = `${JSON.stringify(logMessage)}\n`;
```

Comment:
I got nicked a few times in other code reviews for adding the newline using es6 templates versus appending it. Figured I'd make a note incase that's something we want to be consistent with.